### PR TITLE
Use server's zoom fov for distant world loading.

### PR DESCRIPTION
**Pull request by lhofhansl**
Better fix for #7234

### CODE REVIEW

**Comment by paramat** [comment]
Will look at it now, i just need to remember what is going on and what i did.

**Comment by paramat** [comment]
:+1: but haven't tested.

**Comment by paramat** [comment]
But needs a capital letter.

File: src/clientiface.cpp [hunk 1]
```diff
@@ -196,30 +196,21 @@ void RemoteClient::GetNextBlocks (
 	s16 wanted_range = sao->getWantedRange() + 1;
 	float camera_fov = sao->getFov();
 
-	// If below the heuristic zoom threshold (see adjustDist() in numeric.cpp)
 	// distrust client-sent FOV and get server-set player object property
```
Comment (paramat) [inline src/clientiface.cpp]: Distrust ..

File: src/clientiface.cpp [hunk 2]
```diff
@@ -196,30 +196,21 @@ void RemoteClient::GetNextBlocks (
...
 	// zoom FOV (degrees) as a check to avoid hacked clients using FOV to load
 	// distant world.
 	// 0.888 radians is slightly larger than the zoom threshold of 1.775 / 2
```
Comment (paramat) [inline src/clientiface.cpp]: Comment no longer needed.

**Comment by paramat** [comment]
And a comment needs removing.

**Comment by paramat** [comment]
:+1: once updated.

**Comment by lhofhansl** [comment]
Thanks @paramat will do now (after comment fixes)

**Comment by lhofhansl** [comment]
a1868e8f

**Comment by lhofhansl** [comment]
Hmm... Considering again... Now this is *always* using the server configured zoom fov for distance calculation... A client can now look pretty far - as far as the server's zoom - now without doing anything whatsoever. We want to limit the worst case, but not affect the camera_fov.
I'll do a new PR that fix that.

This is pretty bad. Apologies. @paramat

**Comment by paramat** [comment]
No problem, i missed any issue too.

**Pull request closed without merging**